### PR TITLE
feat: add beta banner for profile launch

### PR DESCRIPTION
**Pull request by nickfranciosi**
adds beta banner with feedback and opt out links based on the profile variant cookie

### CODE REVIEW

File: sass/settings/_colors.scss [hunk 1]
```diff
@@ -12,6 +12,7 @@ $color-orange: #ffc83f !default;
 $color-yellow: #feef6d !default;
 
 $active-color: #0fdebd !default;
+$beta-banner-color: #1bca8f !default;
```
Comment (thomasthesecond) [inline sass/settings/_colors.scss]: In Backpack, we have `accentGreen` as #16c98d, which is nearly identical, but I wonder if we should use it instead?

Comment (thomasthesecond) [inline sass/settings/_colors.scss]: Maybe it doesn't matter here

File: src/components/beta_banner/beta_banner.hbs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<div class="lp-beta-banner">
+  <div class="lp-beta-banner__container">
+      <div class="lp-beta-banner--left">
+        <p>Welcome to our new beta experience! Help us test new functionality. {{test}}</p>
```
Comment (thomasthesecond) [inline src/components/beta_banner/beta_banner.hbs]: Should this `{{test}}` be in here?

Comment (nickfranciosi) [inline src/components/beta_banner/beta_banner.hbs]: nope - good catch!

File: src/components/header/header_component.js [hunk 3]
```diff
@@ -64,6 +70,46 @@ class Header extends Component {
...
+      e.preventDefault();
+      this.leaveBeta();
+    });
+  }
+
+  leaveBeta() {
+    this.cookieUtil.removeCookie("_v"); // Remove Variant
+    window.location = "//connect.lonelyplanet.com/users/sign_out"; // Sign Out
+  }
+
```
Comment (thomasthesecond) [inline src/components/header/header_component.js]: I wonder if all of this logic for the banner should be extracted out into beta_banner/index.js and then imported into the header so it's easier to remove when we're done with it?

I think you would use it like `this.betaBanner = new BetaBanner();`

**Comment by nickfranciosi** [comment]
@thomasthesecond updated

File: src/components/header/header_component.js [hunk 4]
```diff
@@ -2,8 +2,10 @@ import { Component } from "../../core/bane";
 import SearchComponent from "../search";
 import NavigationComponent from "../navigation";
 import NavigationState from "../navigation/navigation_state";
+import CookieUtil from "../../core/cookie_util";
 import $ from "jquery";
 import debounce from "lodash/debounce";
+import BetaBannerComponent from "../beta_banner/beta_banner_component";
```
Comment (thomasthesecond) [inline src/components/header/header_component.js]: I think you can import this into beta_banner/index.js and then just ``mport BetaBannerComponent from "../beta_banner/";

Like we do with https://github.com/lonelyplanet/rizzo-next/blob/master/src/components/footer/index.js

File: src/components/header/index.js [hunk 5]
```diff
@@ -2,6 +2,7 @@ import HeaderComponent from "./header_component";
 
 import "./_header.scss";
 import "../logo/logo.scss";
+import "../beta_banner/_beta_banner.scss";
```
Comment (thomasthesecond) [inline src/components/header/index.js]: I _think_ you can remove this since you're importing into beta_banner/index.js above

Comment (nickfranciosi) [inline src/components/header/index.js]: I am not importing the scss anywhere else so I think it does need to be there, I can import someplace else though

Comment (thomasthesecond) [inline src/components/header/index.js]: You have it here:

https://github.com/lonelyplanet/rizzo-next/pull/718/files#diff-72ac1f525538159e260819c707524fef

But I don't think you're importing that anywhere. But if you look at my comment above and you import the SCSS and beta_banner_component into index.js, then you can just import index.js into header_component.js.

Hope that makes sense

File: src/components/beta_banner/index.js [hunk 6]
```diff
@@ -0,0 +1 @@
+import "./_beta_banner.scss";
```
Comment (thomasthesecond) [inline src/components/beta_banner/index.js]: If you make this:

```
import BetaBanner from "./beta_banner_component";

require("./_beta_banner.scss");

export default BetaBanner;
```

Then in header_component, you can:

```
import BetaBanner from "../beta_banner";
```

and delete `import "../beta_banner/_beta_banner.scss";` from header/index.js

Comment (nickfranciosi) [inline src/components/beta_banner/index.js]: this is actually not working. I don't really see an issue with the banner css being included with the header. Just the same as the navigation , notification and logo scss are being included with the header. This is just another piece of the header.

**Pull request merged**